src/ch01-00-foreword.md

```diff
@@ -0,0 +1,39 @@
+# Foreword
+
+In recent years, it has become an industry standard to deploy software using
+some form of containerization; and while this trend has helped to reduce the
+complexity of our deployments by offering process and file isolation at
+runtime, it has done virtually nothing to reduce the complexity of building
+and maintaining software.
+
+Nix tries to bridge this gap using the same isolation primitizes used by
+container technologies to ensure environmental conditions are precisely
+mirrored _at build time_.
```

Comment:
```suggestion
met _at build time_.
```

Reply:
does "met" clarify that this is a reproducible attribute though?

Reply:
yep, I'd say so. I sense 'mirrored' might convey a special thought that is not immediately inferrable by the reader...

Reply:
less syllables :+1: